wizard/WizardNav.qml

```diff
@@ -89,19 +88,27 @@ GridLayout {
         Layout.fillWidth: true
         color: "transparent"
 
-        RowLayout {
+        PageIndicator {
             id: wizardProgress
-            spacing: 0
-            width: 100  // default, dynamically set later
-            height: 30
+            currentIndex: menuNav.progress
+            count: menuNav.progressSteps
             anchors.horizontalCenter: parent.horizontalCenter
             anchors.verticalCenter: parent.verticalCenter
+            spacing: 25
+            delegate: Rectangle {
+                    implicitWidth: 10
+                    implicitHeight: 10
```

Comment:
Is there any reason to use `implicitHeight`?

Reply:
In Ubuntu, both implictWidth and implicitHeight are required for the PageIndicator component to appear